src/components/MeetOrganizers.jsx

```diff
@@ -0,0 +1,35 @@
+import { ABOUT_HYDPY } from '@/details';
+import Image from 'next/image';
+
+export default function MeetOrganizers() {
+  return (
+    <div className="flex flex-col lg:flex-row justify-center items-center p-6 pt-12 pb-12 border-t-2 border-gray-300">
+      <div className="max-w-3xl lg:mr-20">
+        <h1 className="text-4xl font-bold text-[#001ccf] dark:text-white mb-4">
```

Comment:
- There should be only one h1 in the page. Can you make it h2?
- Can you change the text color to primary/secondary for light mode. The current one doesn't match with the theme.
- Can you make the heading center aligned and above the logo and content?
- Use `dark:text-gray-50`. This is being used everywhere.

Reply:
Updated styling add requested.